examples/kafka_mgmt/get_kafkas_example.py

```diff
@@ -0,0 +1,34 @@
+import dimakis_test_kafka_mgmt_sdk
+from pprint import pprint
+from dimakis_test_kafka_mgmt_sdk.api import default_api
+import sys
+
+# Defining the host is optional and defaults to https://api.openshift.com
+# See configuration.py for a list of all supported configuration parameters.
+configuration = dimakis_test_kafka_mgmt_sdk.Configuration(
+    host = "https://api.openshift.com"
+)
+
+# The client must configure the authentication and authorization parameters
+# in accordance with the API server security policy.
+# Examples for each auth method are provided below, use the example that
+# satisfies your auth use case.
+
+ACCESS_TOKEN = str(sys.argv[1])
+
+# Configure Bearer authorization (JWT): Bearer
+configuration = dimakis_test_kafka_mgmt_sdk.Configuration(
+    access_token = ACCESS_TOKEN,
```

Comment:
Strange to create 2 configuration objects. Can we create single object?

Reply:
ah yes. thats the way it was generated. it is strange, you're right.